packages/workspace/src/browser/workspace-service.ts

```diff
@@ -179,6 +179,25 @@ export class WorkspaceService implements FrontendApplicationContribution {
         return options !== undefined && !!options.preserveWindow;
     }
 
+    /**
+     * Return true if one of the paths in paths array is present in the workspace
+     * NOTE: You should start paths with /
```

Comment:
> You should start

That is not true. See your changes [here](https://github.com/theia-ide/theia/pull/2261/files#diff-b16869c7f6a204b1614c1e58569863cbR46). Please update with this or something similar.

    NOTE: You should always explicitly use `/` as the separator between the path segments.